hawkrest/__init__.py

```diff
@@ -92,11 +115,14 @@ def authenticate(self, request):
         # Pass our receiver object to the middleware so the request header
         # doesn't need to be parsed again.
         request.META['hawk.receiver'] = receiver
-        return on_success
+        return self.hawk_user_lookup(request, receiver.resource.credentials)
 
     def authenticate_header(self, request):
         return 'Hawk'
 
+    def get_user(self, user_id):
+        return HawkAuthenticatedUser()
```

Comment:
Added comment and test; should I mention it in the docs, too, or leave it just for those digging around?